### PR TITLE
(Maybe) Fix broken billing sample

### LINKED ISSUE #1988: functions/billing: test failure preventing PR merges #1985
This test appears to have been failing for at least six months. Latest failure log has this:

    1) "before all" hook in "{root}":
     Error: Invalid Credentials
      at Gaxios._request (node_modules/gaxios/build/src/gaxios.js:89:23)
      at process._tickCallback (internal/process/next_tick.js:68:7)

### PR DESCRIPTION
**Pull request by ace-n**
Fixes #1988

### CODE REVIEW

File: functions/billing/test/index.test.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const sinon = require('sinon');
 const cwd = path.join(__dirname, '..');
 
 const PROJECT_NAME = process.env.GOOGLE_CLOUD_PROJECT;
-
+const FF_CMD = 'functions-framework';
```
Comment (grant) [inline functions/billing/test/index.test.js:26]: In general we should probably use the absolute path to the binary, not the alias. Might cause fewer issues.

**Review by grant — APPROVED** [review]

File: functions/billing/index.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ exports.stopBilling = async (pubsubEvent, context) => {
     return `No project specified`;
   }
 
-  await _setAuthCredential();
+  _setAuthCredential();
```
Comment (grant) [inline functions/billing/index.js:65]: Well, if you're not going to await an async function, you're just asking for issues.

I think you mean to remove `async` from `_setAuthCredential` too.

**Review by grant — CHANGES_REQUESTED** [review]
Found 1 issue we should fix before maybe merging.

File: functions/billing/index.js (hunk 2 above)
Comment (ace-n) [reply functions/billing/index.js:65]: Fixed.

File: functions/billing/test/index.test.js (hunk 1 above)
Comment (ace-n) [reply functions/billing/test/index.test.js:26]: See b/170169197 🙂

**Comment by ace-n** [comment]
@grant please merge for me once approved - thanks!

File: functions/billing/index.js [hunk 3]
```diff
@@ -75,16 +75,11 @@ exports.stopBilling = async (pubsubEvent, context) => {
...
-
-  let client = res.credential;
-  if (client.hasScopes && !client.hasScopes()) {
-    client = client.createScoped([
-      'https://www.googleapis.com/auth/cloud-billing',
-      'https://www.googleapis.com/auth/cloud-platform',
-    ]);
-  }
+const _setAuthCredential = () => {
+  const client = new GoogleAuth({ scopes: [
```
Comment (grant) [inline functions/billing/index.js:79]: I think we should format JSON with normal formatting, like with scopes on a separate line.

**Review by grant — APPROVED** [review]

File: functions/billing/index.js (hunk 3 above)
Comment (ace-n) [reply functions/billing/index.js]: Done.

**Pull request merged**